code/modules/clothing/rogueclothes/pants/pants.dm

```diff
@@ -345,11 +345,11 @@
 	body_parts_covered = GROIN
 
 /obj/item/clothing/under/roguetown/trou/otavan
-	name = "otavan leather trousers"
-	desc = "padded leather armor made by Otavan tailors, its quality is remarkable."
+	name = "fencer leather trousers"
+	desc = "padded leather armor made by the most skilled of kingsfield's tailors, its quality is remarkable."
 	icon_state = "fencerpants"
 	resistance_flags = FIRE_PROOF
-	armor = list("blunt" = 85, "slash" = 75, "stab" = 70, "piercing" = 20, "fire" = 0, "acid" = 0)
+	armor = list("blunt" = 85, "slash" = 75, "stab" = 70, "bullet" = 20, "fire" = 0, "acid" = 0)
 	prevent_crits = list(BCLASS_CUT, BCLASS_STAB, BCLASS_CHOP, BCLASS_BLUNT)
 	blocksound = SOFTHIT
 	max_integrity = 300
```

Comment:
This should probably be at 200 integrity. 300 is way to high to be in line with any other pant armor on the server